projects/arlas-toolkit/src/lib/tools/errors/backend-error.ts

```diff
@@ -32,20 +33,23 @@ export class BackendError extends ArlasError {
     this.hubUrl = hubUrl;
     this.title = marker('ARLAS encountered an error');
     if (this.status === 404) {
+      this.message = message;
       this.showAction = true;
       this.actionMessage = marker('go to arlas hub');
       this.actionType = 'link';
-      this.message = message;
     } else if (this.status === 400) {
       this.message = message;
       this.showAction = true;
       this.actionMessage = marker('go to arlas hub');
       this.actionType = 'link';
     } else if (this.status === 502) {
-      this.message = 'The connection to ' + this.service + ' is lost';
+      this.message = marker('The connection is lost');
```

Comment:
I know this is an already asked question, and I don't remember the answer

BY doing this, we prompt an error dialog each time the backend has an error. 
If I remeber  well this a blocking point for thumbnails that can't be fetched.
Do you remeber what we agreed on this point?

Reply:
There is no popup being displayed for a thumbnail that can't be fetched, or at least I did not experience that behaviour while testing